byterun/sys.c

```diff
@@ -82,8 +82,8 @@ CAMLexport void caml_sys_error(value arg)
   if (arg == NO_ARG) {
     str = caml_copy_string(err);
   } else {
-    int err_len = strlen(err);
-    int arg_len = caml_string_length(arg);
+    mlsize_t err_len = strlen(err);
```

Comment:
Shouldn't this be `size_t`?

Reply:
Disregard that. I'm a crazy person who hates implicit conversions, compiles code with `-Wconversion`, and inserts tons of carefully thought-out casts.

Perhaps doing the same for camlrun would help catch a couple of bugs related to integer overflow or lossy integer undercasts, but that is a huge undertaking, and many people would hate the abundance of casts (some of my colleagues do).